runtime/interpreter/errors.go

```diff
@@ -357,11 +357,21 @@ func (e UUIDUnavailableError) Error() string {
 }
 
 // TypeLoadingError
-
+//
 type TypeLoadingError struct {
 	TypeID common.TypeID
 }
 
 func (e TypeLoadingError) Error() string {
 	return fmt.Sprintf("failed to load type: %s", e.TypeID)
 }
+
+// EncodingUnsupportedValueError
+//
+type EncodingUnsupportedValueError struct {
+	Value Value
+}
+
+func (e EncodingUnsupportedValueError) Error() string {
```

Comment:
Doesn't it need to be of FVM error (so implement `Code() uint32` as well) - so it will be passed as Tx error?

Reply:
This is Cadence, which doesn't know anything about FVM. This PR only makes it easier to look for such an error, without having to resort to looking at the error message.

Reply:
Once this PR is merged I'll update FVM and look for this error